packages/utilities/src/useFocusRects.ts

```diff
@@ -86,9 +108,9 @@ export function useFocusRects(rootRef?: React.RefObject<HTMLElement>): void {
     let onPointerDown: (ev: PointerEvent) => void;
     let onKeyDown: (ev: KeyboardEvent) => void;
     let onKeyUp: (ev: KeyboardEvent) => void;
-    if (providerRef && providerRef.current) {
-      el = providerRef.current;
-      const callbacks = setCallbackMap(el as HTMLElement);
+    if (context?.providerRef.current) {
```

Comment:
>context?.providerRef.current [](http://example.com/codeflow?start=8&length=28)

Should this be `context?.providerRef?.current`?

Reply:
Ah, I see you made `providerRef` no longer optional

Reply:
Yeah I changed it so the context itself is optional, and all of the fields inside are required.